flags/flags.go

```diff
@@ -35,6 +35,9 @@ var (
 	// Full represents the switch to disable all output truncation, passed as a flag.
 	Full bool
 
+	// Name is the name of a cluster or node pool.
+	Name string
```

Comment:
Why is it either or? Oh, and why are these flags even global? It would be nice to have the flags per command. So they are isolated. This is then also what the reference implementation in `devctl` does.

Reply:
I guess you figured it. It's because `flags.Name` is used as the name flag for creating a cluster in one command and the one for naming a node pool in another command.

It may have downsides, but it also has benefits. This way it's actually possible to link all places using, for example, the flag `--aws-instance-type` to a common place (`flags.WorkerAwsEc2InstanceType`).